src/main/java/org/jboss/ejb/client/Affinity.java

```diff
@@ -117,11 +129,24 @@ public String toString() {
     static class LocalAffinity extends Affinity {
 
         private static final long serialVersionUID = -2052559528672779420L;
+        private static final URI uri;
+
+        static {
+            try {
+                uri = new URI("local", "-", null);
```

Comment:
I still don't like this; trying to think of a better way, but if anyone has a suggestion or idea please speak up.